src/scm/gitlab.js

```diff
@@ -110,70 +118,71 @@ class Gitlab {
-      )
-      .then(resolve)
-      .fail(reject)
-    })
-    .then(response => {
-      const re = new RegExp(`(\\${context.config.filetype}|\\.html${context.config.manifestEnabled ? '|^appsscript.json' : ''})$`);
-      const promises = response.filter((tree) => {
-        return tree.type === 'blob' && re.test(tree.path);
+        return $.getJSON(
+            `${this.baseUrl}/projects/${context.repo.id}/repository/tree?ref=${context.branch}&recursive=true&${this.tokenParam}`, {}
+          )
+          .then(resolve)
+          .fail(reject)
       })
-      .map(tree => {
-        return new Promise((resolve, reject) => {
-          $.getJSON(`${this.baseUrl}/projects/${context.repo.id}/repository/files/${encodeURIComponent(tree.path)}?ref=${context.branch}&${this.tokenParam}`, {})
-          .then((content) => {
-            resolve({file: tree.path, content: decodeURIComponent(escape(atob(content.content)))});
+      .then(response => {
+        const re = new RegExp(`(\\${context.config.filetype}|\\.html${context.config.manifestEnabled ? '|^appsscript.json' : ''})$`);
+        const promises = response.filter((tree) => {
+            return tree.type === 'blob' && re.test(tree.path);
           })
-          .fail(reject)
-        });
+          .map(tree => {
+            return new Promise((resolve, reject) => {
+              $.getJSON(`${this.baseUrl}/projects/${context.repo.id}/repository/files/${encodeURIComponent(tree.path)}?ref=${context.branch}&${this.tokenParam}`, {})
+                .then((content) => {
+                  resolve({
+                    file: tree.path,
+                    content: decodeURIComponent(escape(atob(content.content)))
+                  });
+                })
+                .fail(reject)
+            });
+          });
+        return Promise.all(promises);
       });
-      return Promise.all(promises);
-    });
   }
 
   getNamespaces() {
-    return getAllItems(Promise.resolve(
-      {
-        tokenParam: this.tokenParam,
-        items: [],
-        url: `${this.baseUrl}/groups?per_page=25`
-      }),
-      this.followPaginate,
-      'gitlab'
-    )
-    .then(groups => {
-      this.namespaces = [this.user].concat(groups.map(group => group.name));
-      this.namesToIds.groups = groups.reduce((obj, item) => (obj[item.name] = item.id, obj), {});
-      return this.namespaces;
-    })
-    .catch((err) => {
-      showAlert('Failed to get user info.', LEVEL_ERROR);
-    });
+    return getAllItems(Promise.resolve({
+          tokenParam: this.tokenParam,
+          items: [],
+          url: `${this.baseUrl}/groups?per_page=25`
+        }),
+        this.followPaginate,
+        'gitlab'
+      )
+      .then(groups => {
+        this.namespaces = [this.user].concat(groups.map(group => group.name));
+        this.namesToIds.groups = groups.reduce((obj, item) => (obj[item.name] = item.id, obj), {});
+        return this.namespaces;
+      })
+      .catch((err) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/scm/gitlab.js#L162"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>